app/notifications/logbook_entry_updated_notification.rb

```diff
@@ -3,7 +3,7 @@ class LogbookEntryUpdatedNotification < Noticed::Base
   deliver_by :email, mailer: "LogbookEntryMailer", method: "updated_email", if: :email_notifications?
 
   DEFAULT_SEND_NOTIFICATION = true
-  DEFAULT_SEND_EMAIL_NOTIFICATION = true
+  DEFAULT_SEND_EMAIL_NOTIFICATION = false
```

Comment:
I think this will break the RSpec tests? How bad is letting the seeding create these email notifications? Does it make it totally unusable?

Reply:
@hvillero can confirm that this breaks some of the RSpec tests.
- after adding the `password { "password" }` under line 5 of  `factories/user.rb` (this is to fix the validation requiring a user to have a password)
- running `rspec` will cause the following tests to fail:
  -  logbook_entry_updated_notification_spec.rb:14
  - logbook_entry_updated_notification_spec.rb:18
  - new_aircraft_created_notification_spec.rb:14
  - new_aircraft_created_notification_spec.rb:18
  - new_logbook_entry_created_notification_spec.rb:18
  - new_logbook_entry_created_notification_spec.rb:22 
  - user_sign_up_notification_spec.rb:11 
  - user_sign_up_notification_spec.rb:15
  - user_update_notification_spec.rb:11 
  - user_update_notification_spec.rb:15

Reply:
Yep, I will revert this change and run the specs, thx